src/Main.hs

```diff
@@ -1,29 +1,57 @@
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
 
 module Main where
 
 import Control.Concurrent.MVar (takeMVar, newMVar)
-import Data.Maybe              (listToMaybe)
 import Data.MultiSet           (distinctSize)
-import Data.Text               (pack)
-import System.Environment      (getArgs)
+import Data.Text               (Text, pack)
+import Data.Semigroup          ((<>))
 
 import Echidna.Exec
 import Echidna.Solidity
 
 import Hedgehog hiding (checkParallel)
 import Hedgehog.Internal.Property (GroupName(..), PropertyName(..))
 
+import Options.Applicative
```

Comment:
Can we explicitly list imports here?

Reply:
When I started adding explicit imports, there were more than a dozen to add since each of the methods to build the parser info is a separate method. (i.e. fullDesc, progDesc, metavar, ...)